apps/test/unit/templates/teacherDashboard/SectionAssignerTest.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {mount} from 'enzyme';
+import {expect} from '../../../util/configuredChai';
```

Comment:
please use reconfiguredChai, because we are trying to move away from chai-enzyme. your tests look like they do not use any chai-enzyme syntax, but if you do end up needing to update them feel free to ping me.

Reply:
There are tons of examples in many other LP components where we're still using configuredChai.  Are we tracking this clean up anywhere?

Reply:
@islemaster or @jmkulwik , do you have any context on the reason for deprecating `configuredChai` + `chai-enzyme`? I know we wanted to do it at some point, but also that we discovered that it's not necessary in order to upgrade React. so I guess I am looking for confirmation that this is still something we should be moving away from on a best-effort basis (or not).